unlock-js/src/web3Service.js

```diff
@@ -116,6 +119,45 @@ export default class Web3Service extends UnlockService {
         })
       },
     }
+
+    this.ethers_inputsHandlers = {
+      createLock: async (transactionHash, contractAddress, params) => {
+        // The annoying part here is that we do not have the lock address...
+        // Since it is not an argument to the function.
+        // We will 'guess' it again using generateLockAddress
+        // knowing that this may create a race condition another lock creation pending transaction
+        // exists.
+        const newLockAddress = await this.ethers_generateLockAddress()
```

Comment:
Have you noticed any difference of behavior on this vs Web3?

Reply:
no, this is an issue with the chain, until we have `CREATE2` in the smart contract, so ethers is the same, just slightly prettier javascript ;)